src/components/table/column.rs

```diff
@@ -82,14 +89,80 @@ where
         class.push(classes!("pf-m-center"));
     }
 
+    if props.onsort.is_some() {
+        class.push(classes!("pf-v5-c-table__sort"));
+    }
+
     class.extend_from(&props.width);
     class.extend_from(&props.text_modifier);
 
     match &props.label {
         None => html! (<th></th>),
-        Some(label) => html! (
-            <th {class} scope="col" role="columnheader">{ &label }</th>
-        ),
+        Some(label) => {
+            let th_content = if let Some(onsort) = &props.onsort {
+                let header_context = table_header_context.unwrap();
```

Comment:
Unwrapping will cause the application to panic if the context is not found. I agree, that might be programming error, in which case I would still prefer to use `.expect("Some message")` to explain the cause.

Alternatively, it would be possible to simple turn this into a no-op when the context is not found. Both is ok.